samples/secret-santa/secret_santa.ipynb

```diff
@@ -0,0 +1,314 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d25099d2",
+   "metadata": {},
+   "source": [
+    "Copyright (c) Microsoft Corporation.\n",
```

Comment:
We don't need the MIT license bit at the start of the notebook, there's a licence bit included in the main repo